src/scripts/menus.js

```diff
@@ -17,7 +17,7 @@ const isMac = process.platform === 'darwin';
 document.title = APP_NAME;
 
 function getLabel (label) {
-    return isMac ? label : `&${label}`;
+    return i18n.__(isMac ? label : `&${label}`);
```

Comment:
I dont think i18n.__('&Help') would work, so
> return isMac ? i18n.__(label) : `&${i18n.__(label)}`);

Reply:
You mean because History comes first?

Reply:
Turns out mac will just ignore the `&`, so it is safe just to use it anyway. As for the History and Help issue, we will have to specify the `&` somewhere in the middle of the string (e.g. `H&istory`). Should we put this in the translation, or will it be too confusing? - As in some languages they may need to change the letter based on duplicates